docs/templates/toc.yml

```diff
@@ -60,6 +60,8 @@
       name: Indexers
     - name: pandas
       uid: bigframes.pandas
+    - name: Plotting
+      uid: bigframes.operations.plotting
```

Comment:
This adds only `plotting` but not the subclasses underneath. It should be an `- items` entry for its items

Reply:
We only want to add it as subclasses just like `DataFrame`, `pandas`, and `Window` here :https://screenshot.googleplex.com/5UUPmUZZX8RjYAr